common/network-common/src/main/java/org/apache/spark/network/util/AbstractFileRegion.java

```diff
@@ -24,7 +24,7 @@
 
   @Override
   @SuppressWarnings("deprecation")
-  public final long transfered() {
```

Comment:
This should be kept for compatibility I believe.

Reply:
Yeah (I noticed that as I was reviewing -- https://github.com/apache/spark/pull/30323#discussion_r521050706 -- it takes a long time to write the full review)